lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -1211,6 +1211,28 @@ def test_update_thread(self, user, thread_author, commentable_id, status_code, m
         )
         self.assertEqual(response.status_code, status_code)
 
+    @ddt.data(
+        ('student_in_team', 'team_commentable_id', 401),
+        ('student_not_in_team', 'team_commentable_id', 401),
+        ('moderator', 'team_commentable_id', 200)
+    )
+    @ddt.unpack
+    def test_delete_comment(self, user, commentable_id, status_code, mock_request):
+        commentable_id = getattr(self, commentable_id)
+        self._setup_mock(user, mock_request, {"closed": False, "commentable_id": commentable_id})
```

Comment:
If you add "user_id", I think you could make "student_in_team" the author. This would verify that student_in_team can delete her own post.